website/pages/docs/builders/amazon/chroot.mdx

```diff
@@ -127,24 +127,35 @@ Here is a basic example. It is completely valid except for the access keys:
 <Tab heading="HCL2">
 
 ```hcl
+// To make Packer read these variables from the environment into the var object,
+// set the environment variables to have the same name as the declared
+// variables, with the prefix PKR_VAR_.
+
+// You could also hardcode them into the file, but we recommend against that.
+
+// export PKR_VAR_aws_access_key=$YOURKEY
```

Comment:
```suggestion
// There are other idiomatic ways to [set an environment variables](/docs/from-1.5/variables#assigning-values-to-build-variables).

// export PKR_VAR_aws_access_key=$YOURKEY
```

Reply:
( ditto throughout )

Reply:
Tweaking the grammar to be more fluent English, but I'll commit this update.